index.js

```diff
@@ -161,11 +162,11 @@ function createSeries (opts, archs, platforms) {
           })
         }
 
-        if (combination.platform === 'darwin') {
+        if (combination.platform === 'darwin' || combination.platform === 'mas') {
```

Comment:
Yeah, that's better. I should have looked a bit more closely at where it was defined.